newIDE/app/src/AssetStore/ExtensionStore/ExtensionListItem.js

```diff
@@ -81,7 +84,7 @@ export const ExtensionListItem = ({
               {alreadyInstalled && (
                 <Chip
                   size="small"
-                  label={<Trans>Already installed</Trans>}
+                  label={fromStore ? <Trans>Already installed</Trans> : <Trans>Already in project</Trans>}
```

Comment:
Do you think we should use the word "conflict" here? To show that things are coming from a "different background"?

```suggestion
                  label={fromStore ? <Trans>Already installed</Trans> : <Trans>Conflict with extension in project</Trans>}
```

Reply:
We can't bc of compatibility issues with older projects.
Origin didn't exist so i didn't want to break the wording for them with something as scary as a conflict can be.
So i chose a smooth and unconvincing (or even confusing...) wording.

Tbh i don't have many ideas about it. A wording fitting for
- someonse who really downloaded the extension, but on an old project
- someone who only made his extension with a conflicting name

is not easy 😄 

Source : https://github.com/4ian/GDevelop/blob/master/Core/GDCore/Project/EventsFunctionsExtension.h#L167

Reply:
You're very right, great investigation. A wording that is less frightening is a good idea as for now we still have projects that had extensions coming from the store without the origin being properly registered.